product-backend/src/main/java/org/kainos/ea/service/JobRoleValidator.java

```diff
@@ -0,0 +1,23 @@
+package org.kainos.ea.service;
+
+import org.kainos.ea.model.JobRoleRequest;
+
+public class JobRoleValidator {
+
+    public String isValidJobRole(JobRoleRequest jobRole) {
+
+        if (jobRole.getName().length() > 64) {
+            return "Name longer than 64 characters";
+        }
+
+        if (jobRole.getDescription().length() > 2000) {
+            return "Description longer than 2000 characters";
+        }
+
+        if (jobRole.getSharePointLink().length() > 2137) {
+            return "SharePointLink longer than 2137 characters";
+        }
+
+        return null;
```

Comment:
you could use optional here as well no avoid operating on nulls (can be very dangerous!)
```suggestion
public class JobRoleValidator {

    public Optional<String> isValidJobRole(JobRoleRequest jobRole) {

        if (jobRole.getName().length() > 64) {
            return Optional.of("Name longer than 64 characters");
        }

        if (jobRole.getDescription().length() > 2000) {
            return Optional.of("Description longer than 2000 characters");
        }

        if (jobRole.getSharePointLink().length() > 2137) {
            return Optional.of("SharePointLink longer than 2137 characters");
        }

        return Optional.empty();
```